AWS-CloudFormation/s3Resources/marklogic-sink.properties

```diff
@@ -2,105 +2,21 @@
 
 name=marklogic-sink
 connector.class=com.marklogic.kafka.connect.sink.MarkLogicSinkConnector
-
-# Should only need one task since it's using a WriteBatcher, which is multi-threaded
 tasks.max=1
-
 # Topics to consume from [comma separated list for multiple topics]
 topics=marklogic
 
-
 # MarkLogic connector-specific properties
+# See ./config/marklogic-sink.properties for information on each of these
```

Comment:
Simplified this file so we don't need to keep the config docs in sync with the main marklogic-sink.properties file